inc/itilfollowup.class.php

```diff
@@ -1099,9 +1099,10 @@ public static function buildParentCondition(
          );
       }
 
+      $lc_itemtype = strtolower($itemtype);
```

Comment:
Using `$itemtype::getType()` would be sufficient, no need to update all that code.

Reply:
Agreed, i did this change because I noticed that I had both "Ticket" and "ticket" as itemtypes for my followups in my database but these data must come from incorrect API calls.

GLPI should expect the data to be correctly inserted so `$itemtype::getType()` is fine :).

Fixed in 6afd83b.